kore/test/Test/Kore/IndexedModule/Resolvers.hs

```diff
@@ -37,6 +36,18 @@ objectS1 = simpleSort (SortName "s1")
 objectA :: SentenceSymbol Object (TermLike Variable)
 objectA = mkSymbol_ (testId "a") [] objectS1
 
+-- Two variations on a constructor axiom for 'objectA'.
```

Comment:
It would be nice for the names, or the comment, to describe what's different in the two variants. I can't tell by reading.